docs/fastdds/use_cases/rosbag_capture/rosbag_capture.rst

```diff
@@ -100,72 +108,140 @@ generators get called on your IDL files.
      ${idl_files}
    )
 
-Similarly, we need to add the ROS generators to the package.xml file so colcon can keep track of this
-dependency. Add the following to the package.xml file after the ament_cmake build_depend tags:
+Similarly, the ``package.xml`` file should be modified adding the ROS 2 TypeSupport generator dependency.
+Add the following lines to the ``package.xml`` file after the ``buildtool_depend`` tags:
 
 .. code-block:: bash
 
   <buildtool_depend>rosidl_default_generators</buildtool_depend>
   <exec_depend>rosidl_default_runtime</exec_depend>
   <member_of_group>rosidl_interface_packages</member_of_group>
 
-The last step would be to build the package. From the new_typesupport folder, the command to run is:
+The last step would be to build the package.
+Run the following command within the ``fastdds_record_typesupport`` folder:
 
 .. code-block:: bash
 
-   colcon build
+   RMW_IMPLEMENTATION=rmw_fastrtps_cpp colcon build
+
+The build process will create inside the install folder a new ROS 2 overlay with all the required
+libraries and scripts for the ROS 2 applications to use te type defined in the IDL file.
 
-The build process will create inside the install folder a new ROS2 overlay with all the required libraries
-and scripts for ROS2 applications to use your new type.
+Fast DDS Application tuning
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Fast DDS Application required modifications
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+ROS2 adds special tokens to the topic names depending on the ROS subsystem the topic belongs to. More information
+on this topic can be found `on ROS 2 design documentation
+<https://design.ros2.org/articles/topic_and_service_names.html#examples-of-ros-names-to-dds-concepts>`_ .
 
-.. warning::
-    This section is still under work.
+Using the same IDL files used earlier Fast DDS-Gen can generate the required code to handle the new type in
+Fast DDS. The required changes to make so rosbag can see your application are going to be illustrated via the
```

Comment:
```suggestion
Fast DDS.
The Fast DDS application changes required for rosbag2 to communicate with it are going to be illustrated via the
```

Reply:
This suggestion has not been applied. Avoid using `you` in the documentation.

Another suggestion could be:

```suggestion
The changes required in the Fast DDS application so rosbag2 can communicate with it are going to be illustrated via the
```